crates/pico/tests/integration_tests.rs

```diff
@@ -0,0 +1,62 @@
+use std::sync::atomic::{AtomicUsize, Ordering};
+
+use calc::{ast::Program, error::Result, eval::eval, lexer::Lexer, parser::Parser};
+use pico::storage::DefaultStorage;
+use pico_core::{database::Database as Db, source::SourceId};
```

Comment:
looks like we're clobbering the import of `Database` here, is that by accident? I suppose it can be in scope and clobbered, but that's probably unintended!